bundles/remote_services/discovery_common/src/endpoint_discovery_server.c

```diff
@@ -163,11 +163,13 @@ celix_status_t endpointDiscoveryServer_create(discovery_t *discovery,
     char newPort[10];
 
     do {
+        char listeningPorts[50]={0};
         const char *options[] = {
-                "listening_ports", port,
+                "listening_ports", listeningPorts,
                 "num_threads", DEFAULT_SERVER_THREADS,
                 NULL
         };
+        snprintf(listeningPorts,sizeof(listeningPorts),"%s:%s",(*server)->ip, port);
```

Comment:
It should be made clear that why 50 bytes space is enough.

Reply:
I also prefer the usage of `asprintf`. This will use a malloc and require a free, but is IMO better then using a arbitrarily  max number of bytes.

Reply:
Thank you very much for all of your comments